DoSomething/gladiator
# Leaderboard photos

#### What's this PR do?
For each of the top three users, this PR allows admin to choose which of the user's reportback item images will appear in leaderboard emails.

#### How should this be manually tested?
Navigate to a contest. Go to Email Settings. Check "Show Images" for all leaderboard emails that will be testing. Return to contest page and select a competition. Select "Set Photos" for a leaderboard email and enter the fid to feature for specific user. Should check the following test-scenarios:

     1. If Admin leaves fid field empty for user(s) and submits form, the db rows for those/that user(s) should not change and the test email should feature default (most recent) reportback item image for those/that user(s). 

     2. If Admin changes the fid for a user already present in db, the db row associated with that user should change. 

     3. If Admin enters incorrect fid for specific user/reportback, test email should show default reportback item image (the most recent one).

     4. If Admin enters correct fid for specified user/reportback, test email should show the selected reportback item image. 

     5. If Admin enters valid fid for specified user/reportback and for email type 1 but not for email type 2, the test email for email type 2 should not show the reportback item image entered for email type 1. 

#### Any background context you want to provide?
PR composed of two parts: 
1) The process of editing/updating leaderboard_photos db
2) The process of composing leaderboard emails using the reportback item images present in this db. 

#### What are the relevant tickets?
Fixes #333
